docker/release/ubuntu14.04/Dockerfile

```diff
@@ -12,25 +11,22 @@ RUN locale-gen $LANG && update-locale
 # Install dependencies and clean up
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
-        libc6 \
-        libcurl3 \
-        ca-certificates \
-        libgcc1 \
-        libicu52 \
-        libssl1.0.0 \
-        libstdc++6 \
-        libtinfo5 \
-        libunwind8 \
-        libuuid1 \
-        zlib1g \
-        curl \
-        git \
+		apt-utils \
+		ca-certificates \		
+		curl \       
+		apt-transport-https \		
     && rm -rf /var/lib/apt/lists/*
 
-# Install PowerShell package and clean up
-RUN curl -SLO https://github.com/PowerShell/PowerShell/releases/download/$POWERSHELL_RELEASE/$POWERSHELL_PACKAGE \
-    && dpkg -i $POWERSHELL_PACKAGE \
-    && rm $POWERSHELL_PACKAGE
+# Import the public repository GPG keys for Microsoft
+RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+	
+# Register the Microsoft Ubuntu 14.04 repository
+RUN curl https://packages.microsoft.com/config/ubuntu/14.04/prod.list | tee /etc/apt/sources.list.d/microsoft.list
```

Comment:
@andschwa I used 'tee' here, since I wanted to send the output to both the file and stdout.

This is useful when one is building the image interactively 'docker build ...'. The entry being added shows up in stdout of 'docker build'